### PR TITLE
Support level bugfix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30667

## Description
Removed all occurrences of certification as it comes to partners, which solved the issue.

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/base_validator.py [hunk 1]
```diff
@@ -120,14 +119,9 @@ def update_checked_flags_by_support_level(self, file_path):
...
             metadata_json = self.get_metadata_file_content(metadata_path)
             support = metadata_json.get(PACK_METADATA_SUPPORT)
-            certification = metadata_json.get(PACK_METADATA_CERTIFICATION)
 
-            if support == 'partner':
-                if certification is not None and certification != 'certified':
-                    self.add_flag_to_ignore_list(file_path, 'non-certified-partner')
-
-            elif support == 'community':
-                self.add_flag_to_ignore_list(file_path, 'community')
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/hook_validations/base_validator.py:130]: Why did you remove this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -171,6 +171,7 @@
 * Fixed an issue where **generate-docs** was inserting unnecessary escape characters.
 * Fixed an issue in the **update-release-notes** command where changes to the pack_metadata were not detected.
 * Fixed an issue where **validate** did not check for missing release notes in old format files.
+* Fixed an issue where **validate** command asked for tests for non xsoar supported packs.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:174]:
```suggestion
* Fixed an issue where **validate** command failed when no tests were given for a partner supported pack.
```

File: demisto_sdk/commands/common/hook_validations/base_validator.py (hunk 1 above)
Comment (guyfreund) [reply demisto_sdk/commands/common/hook_validations/base_validator.py:130]: no need for the elif case, packed all in the if case

Comment (ShahafBenYakir) [reply demisto_sdk/commands/common/hook_validations/base_validator.py:130]: True, missed the 'community' reference. 👍

**Review by ShahafBenYakir — APPROVED** [review]

**Review by roysagi — APPROVED** [review]
